data/transactions/logic/doc.go

```diff
@@ -325,7 +324,7 @@ var txnFieldDocs = map[string]string{
 	"Sender":                   "32 byte address",
 	"Fee":                      "micro-Algos",
 	"FirstValid":               "round number",
-	"FirstValidTime":           "Causes program to fail; reserved for future use",
+	"FirstValidTime":           "timestamp of FirstValid-1 block. Panics if used before Teal V5",
```

Comment:
Capitalize Timestamp. And I think we can skip the panic sentence because the autogenerated docs will mention LogicSigVersion >= 5.